### PR TITLE
resource_control: fix resource control usage restriction

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
@nolouch PTAL

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: tidb-resource-control.md [hunk 1]
```diff
@@ -25,10 +25,7 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 
 ## 使用限制
 
-目前，资源管控特性具有以下限制:
-
-- 只支持对前台客户发起的读写请求做限流和调度，不支持对 DDL 以及 Auto Analyze 等后台任务的限流和调度。
-- 资源管控将带来额外的调度开销。因此，开启该特性后，性能可能会有轻微下降。
+资源管控将带来额外的调度开销（约 5%）。因此，开启该特性后，性能可能会有轻微下降。
```
Comment (nolouch) [inline tidb-resource-control.md:28]:
```suggestion
资源管控将带来额外的调度开销。因此，开启该特性后，性能可能会有轻微下降 (<%5)。
```

**Review by nolouch — APPROVED** [review]

File: tidb-resource-control.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ TiDB 资源管控特性提供了两层资源管理能力，包括在 TiDB 层的
 
 ## 使用限制
 
-资源管控将带来额外的调度开销（约 5%）。因此，开启该特性后，性能可能会有轻微下降。
+资源管控将带来额外的调度开销。因此，开启该特性后，性能可能会有轻微下降 (<%5)。
```
Comment (Oreoxmt) [inline tidb-resource-control.md:28]:
```suggestion
资源管控将带来额外的调度开销。因此，开启该特性后，性能可能会有轻微下降（低于 5%）。
```

**Pull request merged**